### PR TITLE
NUX 2.0

### PR DESCRIPTION
**Pull request by bago2k4**
Cards:
- NUX: https://trello.com/c/8XmeTVtv
- Login restyle: https://trello.com/c/MieYdhH5

To test:
- login with a normal user, do you NOT see the NUX? good
- invite a user, do you NOT see the NUX when you signup? good
- signup as a brand new user (no email domain or already used slack org), do you see the NUX? good
- logout and log back in, do you NOT see the NUX? good
- check that the NUX is still showing as expected on different screens (large screen and laptop screen mostly)

### CODE REVIEW

**Comment by bago2k4** [comment]
[NUX 2.0 FTW!](https://trello.com/c/8XmeTVtv/1106-nux-20-ftw)

File: src/oc/web/actions.cljs [hunk 1]
```diff
@@ -1585,13 +1601,10 @@
       (api/get-auth-settings)))
   db)
 
-(defmethod dispatcher/action :onboard-overlay-show
+(defmethod dispatcher/action :nux-end
```
Comment (thepug) [inline src/oc/web/actions.cljs]: `:nux-end` needs to be documented in the README

**Comment by bago2k4** [comment]
[Login popover renew](https://trello.com/c/MieYdhH5/1107-login-popover-renew)

File: resources/public/js/static-js.js [hunk 2]
```diff
@@ -175,4 +177,22 @@ function siteMobileMenuToggle(){
     body.querySelector("div.site-mobile-menu").classList.remove("hidden");
     body.classList.add(menuClass);
   }
+}
+
+// Check nux cookie to see if we need to show the blue setup screen or the regular loading screen
+var nux_cookie = OCStaticGetCookie(OCStaticCookieName("nux"));
```
Comment (thepug) [inline resources/public/js/static-js.js]: The code here can be changed to the following:

```javascript
var nux_cookie = OCStaticGetCookie(OCStaticCookieName("nux"));
var oc_loading = document.querySelectorAll("div.oc-loading");
if (oc_loading) {
  oc_loading.forEach(function(item) {
    var fn = nux_cookie ? 'add' : 'remove';
    item.classList[fn].call('setup-screen');
  });
}
```

File: src/oc/web/components/dashboard_layout.cljs [hunk 3]
```diff
@@ -150,44 +119,89 @@
         topics (:topics board-data)]
       ;; Topic list
       [:div.dashboard-layout.group
-        (when (and @(::show-boards-tooltip s)
-                   (not show-onboard-overlay))
-          (when-let* [nav-boards (js/$ "h3#navigation-sidebar-boards")
-                      offset (.offset nav-boards)
-                      boards-left (aget offset "left")]
+        (when (= nux :3)
```
Comment (thepug) [inline src/oc/web/components/dashboard_layout.cljs]: I'd prefer the nux bits in its own component just to reduce the amount of code to read in `dashboard-layout`.

**Comment by bago2k4** [comment]
@thepug changes pushed

**Pull request merged**